### ISSUE TITLE
Kernel Update

### ISSUE THREAD

**Issue by superna9999** [intel/iotg-yocto-bsp-public#4]
Seems a good idea to drop Linux 4.1 and switch to a longterm kernel like 4.9 or 4.14 with Meltdown and Spectre fixes ?

**Comment by aaronovz1** [comment]
I agree, and have asked an Intel representative about this yesterday. Hoping to hear back soon on what plans they have in the pipeline. It would also be nice to move to a newer version of Yocto, as Jethro branch is pretty old.

**Comment by kimhoemo** [comment]
Thanks for the feedback. Please check with Intel representative through IPS on latest roadmap.

**Comment by aaronovz1** [comment]
Any information that can be provided here? Is MR4 going to be ready this month?

**Comment by kimhoemo** [comment]
Please refer Intel website for latest announcement or release. Alternatively, please check with Intel representative through IPS.
Thank you.

**Comment by aaronovz1** [comment]
I have looked at Intel website and have created an IPS case. Still no answers.
